README.md

```diff
@@ -1,51 +1,118 @@
-and store Zed data, including:
+And rather than having to set up complex ETL pipelines with brittle
+transformation logic, managing your data lake should be as easy as `git`.
 
-* The [`zed`](cmd/zed/README.md) command line tool for managing, searching, and querying a Zed lake
-* The [Zed language](docs/language/README.md) documentation
-* The [Zed formats](docs/formats/README.md) specifications and documentation
+And while _schemas_ are a great way to model and organize your data, they often
+[get in the way](https://github.com/brimdata/sharkfest-21#schemas-a-double-edged-sword)
+when you are just trying to store or transmit your semi-structured data.
 
-The previously released [`zq`](cmd/zed/README.md#zq) tool is now packaged as
-a command-line shortcut for the `zed query` command.
+Finally, we believe a lightweight data store that provides easy search and analytics
+would be a great place to store data sets for data science and
+data engineering experiments running in Python and providing easy
+integration with your favorite Python libraries.
 
-## Installation
+## How?
 
-To install `zed` or any other tool from this repo, you can either clone the repo
- and compile from source, or use a pre-compiled
- [release](https://github.com/brimdata/zed/releases), available for Windows, macOS, and Linux.
+Zed solves all these problems with a new format called
+[ZSON](docs/formats/zson.md),
+which is a superset of JSON and the relational models.
+ZSON is syntax-compatible with JSON
+but it has a comprehensive type system that you can use as little or as much as you like.
+Zed types can be used as schemas.
 
-If you don't have Go installed, download and install it from the
-[Go downloads page](https://golang.org/dl/). Go version 1.16 or later is
-required.
+Zed also has a cloud-based object design that was modeled after
+the `git` design pattern.  Commits to the lake are transactional
+and consistent.  Search index updates are also transactionally
+consistent with any ingested data and searches can run with or
+without indexes.
 
-To install the binaries in `$GOPATH/bin`, clone this repo and
-execute `make install`:
+## Quick Start
 
+_Detailed documentation [is available](docs/README.md)._
+
+The quickest way to get running on macOS, Linux, or windows
+is to download a pre-built release binary.
+You can find these binaries on the Github releases page.
+
+Once installed, you can run the query engine from the command-line using `zq`:
+```
+echo '{"s":"hello, word"}' | zq -Z -
+```
+Or you can run a Zed lake server, load it with data using `zapi`, and hit the API.
+In one shell, run the server
+```
+mkdir scratch
+cd scratch
+zed lake serve
+```
+And in another shell, run the client:
+```
+zapi create Demo
+zapi use Demo@main
+echo '{s:"hello, world"}' | zapi load -
+zapi query "from Demo"
+```
+You can also use `zed` from Python.  After you install the Zed Python:
+```
+pip install "git+https://github.com/brimdata/zed#subdirectory=python/zed"
```

Comment:
```suggestion
pip3 install "git+https://github.com/brimdata/zed#subdirectory=python/zed"
```

I still don't have a great handle on how the world is finalizing the transition from Python2 to Python3, but on my Mac for instance a regular reference to `pip` still tries to use the one for Python2, hence failure:

```
zed requires Python '>=3.3' but the running Python is 2.7.16
```